assets/src/edit-story/components/library/libraryTabs.js

```diff
@@ -68,7 +68,7 @@ function LibraryTabs() {
           key={id}
           id={getTabId(id)}
           isActive={tab === id}
-          onClick={() => setTab(id)}
+          onClick={(evt) => setTab(id, evt)}
```

Comment:
Could we perhaps do it here as:

```js
          onClick={blurOnPointerClick(() => setTab(id)}
```

And then don't change `setTab` or `LibraryProvider` at all - simply change `blurOnPointerClick` to something like:

```js
const blurOnPointerClick = (handler) => (evt) => {
  // invoke handler(evt)
  // then blur if need be
};
```

Reply:
Great idea.